src/pages/Facility/settings/organizations/FacilityOrganizationUsers.tsx

```diff
@@ -152,24 +154,40 @@ export default function FacilityOrganizationUsers({ id, facilityId }: Props) {
                                 {t("phone_number")}
                               </div>
                               <div className="font-medium truncate">
-                                {userRole.user.phone_number}
+                                {userRole.user.phone_number
+                                  ? formatPhoneNumberIntl(
+                                      userRole.user.phone_number,
+                                    )
+                                  : "-"}
                               </div>
                             </div>
                           </div>
                         </div>
                       </div>
 
-                      <div className="mt-2 -mx-2 -mb-2 sm:-mx-4 sm:-mb-4 rounded-md py-4 px-4 bg-gray-50 flex justify-end">
+                      <div className="mt-2 -mx-2 -mb-2 sm:-mx-4 sm:-mb-4 rounded-md py-4 px-4 bg-gray-50 flex justify-end gap-2">
                         <EditFacilityUserRoleSheet
                           facilityId={facilityId}
                           organizationId={id}
                           userRole={userRole}
                           trigger={
                             <Button variant="outline" size="sm">
-                              <span>{t("see_details")}</span>
+                              <span>{t("edit_role")}</span>
                             </Button>
                           }
```

Comment:
we might need to do a permission check right @Jacobjeevan ?

Reply:
yep; I'll add it in the permissions PR once this is merged.

I gotta add it for both locations and depts as well.